src/components/sound.js

```diff
@@ -64,17 +64,29 @@ module.exports.Component = registerComponent('sound', {
     }
     // All sound values set. Load in `src`.
     if (srcChanged) {
-      var self = this;
+      var isStream = (data.src.srcObject instanceof MediaStream);
```

Comment:
if `MediaStream` isn't available or enabled (such as in release-channel Safari desktop or mobile < iOS 11), this will throw a `ReferenceError`:

```
ReferenceError: Can't find variable: MediaStream
```

you can change this to

```js
var isStream = (data.src.srcObject instanceof window.MediaStream);
```

or this:

```js
var isStream = ('MediaStream' in window && data.src.srcObject instanceof MediaStream);
```

if you decide to use `MediaStream` (instead of `window.MediaStream`), you'll probably want to denote the globals in comments:

```js
/* global MediaStream */
```

Reply:
@delapuente friendly ping on this